services/core-api/app/api/verifiable_credentials/resources/w3c_map_credential_resource.py

```diff
@@ -103,3 +103,40 @@ def post(self):
             "credential signed by did:indy, not by did:web and using deprecated acapy endpoints" +
             dumps(signed_credential))
         return signed_credential["signed_doc"]
+
+
+class W3CCredentialUNTPResource(Resource, UserMixin):
+    parser = reqparse.RequestParser(trim=True)
+    parser.add_argument(
+        'permit_amendment_guid',
+        type=str,
+        help='GUID of the permit amendment.',
+        location='json',
+        store_missing=False)
+
+    @api.expect(parser)
+    @api.doc(
+        description=
+        "returns a signed w3c credential for a specific permit_amendment using deprecated aca-py endpoint, but with did:indy:bcovrin:test:"
+    )
+    @requires_any_of([EDIT_PARTY, MINESPACE_PROPONENT])
+    def post(self):
+        if not is_feature_enabled(Feature.JSONLD_MINES_ACT_PERMIT):
+            raise NotImplementedError("This feature is not enabled.")
+        current_app.logger.warning("untp endpoint")
+
+        data = self.parser.parse_args()
+        permit_amendment = PermitAmendment.find_by_permit_amendment_guid(
+            data["permit_amendment_guid"])
+        if not permit_amendment:
+            raise BadRequest("Permit amendment not found")
```

Comment:
should this be a BadRequest or a NotFound? I'd lean towards NotFound. 🤷‍♀️

Reply:
NotFound / 404 is for "Resource Not Found". 

If the permit_amendment_guid was in the URL, it would be 404, but given it's a payload processing thing, i believe 400 is more correct. 

Should permit_amdendment_guid be a URL parameter... maybe.